Sources/StringsLintFramework/Models/MasterRuleList.swift

```diff
@@ -6,9 +6,7 @@
 //
 
 public let masterRuleList = RuleList(rules: [
-    MissingRule.self,
-    UnusedRule.self,
-    PartialRule.self,
```

Comment:
BTW removed all of these since they are not relevant to us thanks to SwiftGen! @harkmall